hack/update-gofmt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# script to run gofmt over our code (not vendor)
+set -o errexit
+set -o nounset
+set -o pipefail
+
+find . -name "*.go" | grep -v "\\/vendor\\/" | xargs gofmt -s -w
```

Comment:
Please don't add a hack/ directory or any shell scripts to this pristine repo? :)

Reply:
We should be able to get travis or whatever to run gofmt directly.